mkdocs.yml

```diff
@@ -24,4 +24,5 @@ nav:
     - Developers: 
         - 'Overview': 'wiki/99 - Developers/README.md'
         - 'SQL Dictionary': 'wiki/99 - Developers/SQL Dictionary.md'
+        - 'Documentation Style Guide': 'HelpStyleGuide.md'
```

Comment:
```suggestion
        - 'Help Style Guide': 'HelpStyleGuide.md'
```

The style guide only applies to help text.

Reply:
```suggestion
        - 'Style Guide (for Module help text)': 'HelpStyleGuide.md'
```

I think this is a bit more clear (though maybe wordy). Thoughts?

Reply:
you don't need the word "Module" in the parenthesis (but it might still be wordy if you remove it)